docs/source/nlp/intro.rst

```diff
@@ -2,10 +2,26 @@ Natural Language Processing (NLP)
 =================================
 
 .. toctree::
-   :maxdepth: 8
+   :maxdepth: 2
 
    models
+   megatron_finetuning
    api
 
 
-NLP tutorials can be found under ``<NeMo_Git_root>/tutorials/nlp/``
+Scripts for running NLP models, could be found under ``NeMo/example/nlp/``.
```

Comment:
Aren't these refs to examples and tutorials duplicates to the following refs?

Should we have a list of supported models and tutorials here? with links to the examples/tutorials and their example script?